docs/outline.md

```diff
@@ -0,0 +1,57 @@
+# User Provisioning
+
+This is the prelimary outline for the fullblown tutorial. This outline will eventually be removed once the tutorial is completed.
+
+## Introduction
+
+1. What we are trying to accomplish
+2. Architectural Overview
+   - Authenticating the script
+   - Creating the users
+   - Assign the user to a group, roles and sites
+   - Creating the WebRTC phones
+3. Pre-requisites
+   Node 11.
+4. Installing the code
+   NPM install
+   Docker install
+
+## Module 1: How the script runs and authenticates the user
```

Comment:
General style commentary: particularly in titles, be concise and avoid passive voice. It's often a good pattern to start with an "ing" verb. A few examples from this page:

* "How the script runs and authenticates the user" > "Running the script and user authentication"
* "What is the relationship between phone bases -> phones and stations" > "Relating phones, phone bases, and stations.
* "What is a group" > "Understanding groups"

Reply:
Changed the titles but most of this is just a brain dump for the outline I want to write.